src/constants/common.rs

```diff
@@ -0,0 +1,18 @@
+use serenity::model::prelude::ChannelId;
+
+lazy_static! {
+  pub static ref CODEFLOW_SUPERVISOR_URL: String = std::env::var("CODEFLOW_SUPERVISOR_URL")
+    .expect("missing CODEFLOW_SUPERVISOR_URL env variables");
+  pub static ref TWO_FACTOR_DEPLOYMENT_CHANNEL: ChannelId = ChannelId(
```

Comment:
Ok @GreeFine no problem if you want a raw value instead of a env variable but should we create a discord channel for deployment ? 
(For now I will use the #Piou channel for dev & prod)